Winget-AutoUpdate-Install.ps1

```diff
@@ -118,6 +118,22 @@ param(
 
 <# FUNCTIONS #>
 
+#Function to get the latest WinGet available version on Github
+Function Get-AvailableWinGetVersion {
+
+    #Get latest WinGet info
+    $WinGeturl = 'https://api.github.com/repos/microsoft/winget-cli/releases/latest'
+
+    try {
+        #Return latest version
+        return ((Invoke-WebRequest $WinGeturl -UseBasicParsing | ConvertFrom-Json)[0].tag_name).Replace("v", "")
+    }
+    catch {
+        return $false
+    }
+
+}
+

```

Comment:
I mean here

Reply:
Yes, but the import of functions doesn't exist in `Winget-AutoUpdate-Install.ps1`, hence the Function inline...
...or do I not understand?

Reply:
Ok, forgot about that. I'll fix it when  I get home

Reply:
it's better, I think. Thanks